tin/apps/assignments/models.py

```diff
@@ -172,6 +173,16 @@ class Assignment(models.Model):
         validators=[MinValueValidator(10)],
     )
 
+    use_submission_cap = models.BooleanField(default=False)
+    submission_cap = models.PositiveSmallIntegerField(
+        null=True,
+        validators=[MinValueValidator(1)],
+    )
+    submission_cap_after_due = models.PositiveSmallIntegerField(
+        null=True,
+        validators=[MinValueValidator(1)],
+    )
```

Comment:
Would be nice to have default values (e.g. 15 and 0) for both caps at the assignment level - this would also improve validation logic and make it consistent with validation for stuff like quiz action and grader timeout